client/Questions&Answers/QAview.jsx

```diff
@@ -13,36 +12,31 @@ class QAview extends React.Component {
       questionsView: false,
       answersView: false,
     };
-    this.handleQuestionAdd = this.handleQuestionAdd.bind(this)
-    this.handleAnswerAdd = this.handleAnswerAdd.bind(this)
+    this.handleInput = this.handleInput.bind(this)
   }
 
-
-  handleAnswerAdd(e, input) {
-    const { postInput } = this.props
-    event.preventDefault()
-    postInput('answer', input)
-    console.log('handle answer add triggered')
-  }
-
-  handleQuestionAdd(e, input) {
+  handleInput(type, id, input) {
     const { postInput } = this.props
-    event.preventDefault()
-    postInput('question', input)
-    console.log('handle question add triggered')
+    // remove after answer modal
+    console.log('handle input triggered', type, id, input)
```

Comment:
It's fine for now, but we should probably remove console.log statements at some point before deployment.